patches/net-dns-dns_transaction.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
+index c12f1c728de558b236a0144179bfa268da90fb04..685ec6c86d16f480dd4a995b0fb64c01853a5113 100644
+--- a/net/dns/dns_transaction.cc
++++ b/net/dns/dns_transaction.cc
+@@ -405,8 +405,7 @@ class DnsHTTPAttempt : public DnsAttempt, public URLRequest::Delegate {
+     request_->SetExtraRequestHeaders(extra_request_headers);
+     // Disable secure DNS for any DoH server hostname lookups to avoid deadlock.
+     request_->SetDisableSecureDns(true);
+-    request_->SetLoadFlags(request_->load_flags() | LOAD_DISABLE_CACHE |
```

Comment:
I think you can avoid this patch by defining `LOAD_BYPASS_PROXY` as `LOAD_DISABLE_CACHE`. If that doesn't work for some reason you should use a define/chromium_src override after the original call instead of changing it to minimize the patch

Reply:
fixed